app/src/main/java/com/conupods/OutdoorMaps/View/SearchView/CampusLocationsAdapterListener.java

```diff
@@ -0,0 +1,8 @@
+package com.conupods.OutdoorMaps.View.SearchView;
+
+import com.conupods.OutdoorMaps.Models.Building.AbstractCampusLocation;
+
+public interface CampusLocationsAdapterListener {
+
+    void onCampusLocationSelected(AbstractCampusLocation contact);
```

Comment:
this seems wonky. have a space above and below or neither would look cleaner

Reply:
that's the android studio reformatter that ran on every final. Unfortunately, i probably shouldn't change it in order to adhere to the convention